test/utils/padder/padder.go

```diff
@@ -61,6 +62,13 @@ type Padder struct {
 	*padRequest
 }
 
+// PaddingOptions is some configuration that modifies options for a pad request.
+type PaddingOptions struct {
+	// LabelSelector pad nodes with a given label.
+	// the number of padded nodes is still limit by padRequest.nNodes value
```

Comment:
which is kinda redundant with the label selector, then.
No need to change this now, but we will need to revisit the API later.

Reply:
ok, not urgent anyway.